homeassistant/helpers/service.py

```diff
@@ -2,6 +2,10 @@
 import functools
 import logging
 
+# pylint: disable=unused-import,ungrouped-imports
+from typing import Optional  # NOQA
+from homeassistant.helpers.typing import HomeAssistantType  # NOQA
```

Comment:
So Home Assistant is optional a lot in the main package, but never in the components. I think that we don't need an optional type.